SupersonicSound/Wrapper/Util.cs

```diff
@@ -183,7 +183,8 @@ public static GUID ToFmod(this Guid guid)
             byte[] last8 = new byte[8];
             Array.Copy(bytes, 8, last8, 0, 8);
 
-            return new GUID {
+            return new GUID
```

Comment:
My VS was just re-formatting this when I added some other code, no real change here.